yadm

```diff
@@ -540,6 +540,7 @@ function move_file() {
 
   mv -f "$temp_file" "$output"
   copy_perms "$input" "$output"
+  [ "$(config --bool yadm.template-read-only)" == "true" ] && chmod -w "$output"
```

Comment:
@soraxas Is the intention here to ensure the write permission is disabled for all? I might be wrong, but I think that specifying `-w` will work different depending on the umask of the user running the command. Should this be `a-w`?

Reply:
Hi @TheLocehiliosan thanks for your response. Ah yep that's true, I reckon `a-w` will be a more correct approach with fewer surprises, thanks

Reply:
Hi @TheLocehiliosan yep you are right, I also agree that perhaps using `a-w` would be the more correct approach